cg-spec/editors_draft.html

```diff
@@ -3059,6 +3059,32 @@ <h2>RDF-star vocabulary</h2>
       This issue proposes to introduce a vocabulary for locally overriding the <a href="#ref-opacity">opaque</a> semantics of <a>quoted triples</a>, making them transparent in some contexts.
     </div>
 
+    <div class="defn">
+      <pre>## Vocabulary for concepts used by or introduced by RDF-star.
+PREFIX rdf:       &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt;
+PREFIX rdfs:      &lt;http://www.w3.org/2000/01/rdf-schema#&gt;
+PREFIX rdf-star:  &lt;http://www.w3.org/ns/rdf-star#&gt;
+
+rdf-star:Triple      a rdfs:Class ;
+    rdfs:isDefinedBy   &lt;http://www.w3.org/ns/rdf-star#&gt;
+    rdfs:label         "RDF-star Triple" ;
+    rdfs:comment       "The class of RDF-star triples." ;
```

Comment:
Is this the class of quoted triples, triples containing a quoted triple, or the superset of all RDF triples which may or may not contain an embedded triple? If the latter, should there be a class for quoted triples?

Reply:
It is the class of triples in RDF-star. A triple in [RDF 1.1 concepts](https://www.w3.org/TR/rdf11-concepts/#section-triples) is three components. The three components define the triple. It can be in more than one graph. Asserted => element of a graph set ; in RDFstar, quoted => may also in an RDF term.

I wondered about a class of quoted triples and came to conclusion that a triple can be quoted or asserted but it exists regardless.

Also, if a quoted triple is different then there is a 1-1 correspondence between quoted and assertable triples. i.e. it makes little difference.

Reply:
I had asked myself the same question and came to the same conclusion as @afs. There is no need to add a class for quoted triples. An RDF-star triple is unique identified by its three components, no matter where it appears.

Reply:
I tried parsing @afs answer to @gkellogg but I didn't succeed because of the mixture of apparent typos, unfinished sentences and probably also some ignorance on my side. Especially this here I'm unable to parse: `in RDFstar, quoted => may also in an RDF term` , but also this: `Also, if a quoted triple is different then there is a 1-1 correspondence between quoted and assertable triples. i.e. it makes little difference`. Could you please again explain?

Reply:
"if a quoted triple is different (from an asserted triple) then there is an isomorphism between quoted and assertable triples"

Reply:
On my book all quoted triples are assertable but not all assertable triples are quoted because a quoted triple requires the act of quoting a (assertable, not necessarily asserted) triple. Quoting a triple is an act that creates something that in RDF-star syntax looks like << s p o >>. This we call quoted triple, formerly we called it embedded triple. I think this needs a class and it is distinct from all triples that are assertable.

Reply:
Or maybe you mean not quoted, but "quotable", the class of all triples that can be quoted? That is indeed isomorphic to the class of all assertable triples. Which brings me back to my question below: what made you choose these terms and not others? What are the use cases?

Reply:
> all quoted triples are assertable
Surely it's "all triples are assertable"? One does not asserted a quoted triple. I think this is what your second comment says? 

There are triples.

They can be used as quoted (RDF terms) or asserted (elements of a set of triples making up a graph) or both.

["RDF-star introduces quoted triple, which is a new kind of RDF term."](https://w3c.github.io/rdf-star/cg-spec/editors_draft.html#quoted-triples).

Reply:
Ok, now I understand.